docs/source/api/v2/staticdnsentries.rst

```diff
@@ -133,7 +133,7 @@ Creates a new, static DNS entry.
 
 Request Structure
 -----------------
-:address:      If ``typeId`` identifies a ``CNAME`` type record, this is the Canonical Name (CNAME) of the server, otherwise it is the IP address to which ``host`` shall be resolved
+:address:      If ``typeId`` identifies a ``CNAME`` type record, this is the Canonical Name (CNAME) of the server that is required to end with a trailing period (eg:cdn.test.com. is the right usage but cdn.test.com will throw an error), otherwise it is the IP address to which ``host`` shall be resolved
```

Comment:
this description seems different than the rest. i think i like the simpler descriptions without the example as i think it's obvious.

Reply:
Ok!